cmd/res/configuration.toml

```diff
@@ -0,0 +1,104 @@
+[Writable]
+LogLevel = "INFO"
+  # Example InsecureSecrets configuration that simulates SecretStore for when EDGEX_SECURITY_SECRET_STORE=false
+  # InsecureSecrets are required for when Redis is used for message bus
+  [Writable.InsecureSecrets]
+    [Writable.InsecureSecrets.DB]
+    path = "redisdb"
+      [Writable.InsecureSecrets.DB.Secrets]
+    [Writable.InsecureSecrets.Camera001]
+    path = "credentials001"
+      [Writable.InsecureSecrets.Camera001.Secrets]
+      username = "administrator"
+      password = "Password1!"
+    # If having more than one camera, uncomment the following config settings
+    [Writable.InsecureSecrets.Camera002]
+    path = "credentials002"
+      [Writable.InsecureSecrets.Camera002.Secrets]
+      username = "administrator"
+      password = "Password1!"
+
+[Service]
+HealthCheckInterval = "10s"
+Host = "0.0.0.0"
+Port = 59985
```

Comment:
Change to port assigned by @iain-anderson

Reply:
Changed the service port to 59984